### PR TITLE
Expand type and docs

### PR DESCRIPTION
**Pull request by whscullin**
Brings our types up to date with Looker 7.14. Adds more detailed comments to demo app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -269,3 +269,21 @@ export const lookId = 1
 * The server will print out what host and port it is running on.
 
 You may need to `pip install six` to install the Python 2/3 compatibility layer.
+
+### Troubleshooting
+
+## Logging
+
+The Embed SDK is built on top of [chatty](https://github.com/looker-open-source/chatty). Chatty users [debug](https://github.com/visionmedia/debug) for logging. You can enable logging
```
Comment (bryans99) [inline README.md:277]: Tested this out for extensions but does not work with a sandboxed IFRAME

File: README.md [hunk 2]
```diff
@@ -269,3 +269,21 @@ export const lookId = 1
 * The server will print out what host and port it is running on.
 
 You may need to `pip install six` to install the Python 2/3 compatibility layer.
+
+### Troubleshooting
```
Comment (erin-docs) [inline README.md:273]: @whscullin Questions for updating docs:

1. Should Logging (currently H2) be a subheading of Troubleshooting (currently H3)?

2. In Docs we have a separate page for [SDK Intro](https://docs.looker.com/reference/embed-sdk/embed-sdk-intro) and [SDK demo](https://docs.looker.com/reference/embed-sdk/embed-sdk-demo) I'm going to chuck your new Logging section on the intro page, would you agree that it's a generic section for SDK and not specific to the demo?

Comment (whscullin) [reply README.md:273]: 1. Markdown - how does it work?

2. Definitely generic.

Comment (erin-docs) [reply README.md:273]: Also, I got busted by our editor while copying your work:

image


Would that be a command?  Is that update correct?

Comment (whscullin) [reply README.md:273]: I guess that works. Programmatically speaking it's a statement, I think, but it's probably more clearly conveyed as a command. Perhaps why I just avoided taxonomy.

Comment (erin-docs) [reply README.md:273]: I was also trying to avoid, but I got called out. I guess some people think it's my "job" to "be clear when writing."

Comment (jkaster) [reply README.md:273]: The temerity of some people

Comment (erin-docs) [reply README.md:273]: The gall.

**Pull request merged**